cmake/projects/restinio/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 Niall Douglas https://www.nedproductions.biz/
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    restinio
+    VERSION
+    0.6.14
+    URL
+    "https://github.com/ned14/restinio/archive/refs/heads/master.tar.gz"  ## To be replaced with the cpp-pm branch
```

Comment:
the downloaded package must not change, otherwise the SHA1 sum will be changed breaking this package. Link to a tag instead.

Reply:
As the comment says, this is temporary. I was seeking help with why all the unit tests are failing.